hypervisor/include/arch/x86/trusty.h

```diff
@@ -126,13 +128,46 @@ struct secure_world_memory {
 struct secure_world_control {
 	/* Whether secure world is enabled for current VM */
 	bool sworld_enabled;
+	/* Secure World's running status, 1:active, 0:inactive */
+	uint32_t sworld_active;
```

Comment:
how about just using one single field "sworld_state" instead of sworld_active?
active,
suspend,
.....

because i'm wondering if we can remove this bit : "sworld_save_area.flag.bits.valid"

Reply:
@kwang13 what's your opinion? I am OK with both solution. 
another question: whether secure_world_control be cleared when S3?